autogpt_platform/backend/backend/blocks/ai_shortform_video_block.py

```diff
@@ -0,0 +1,306 @@
+        )
+        background_music: AudioTrack = SchemaField(
+            description="Background music track",
+            default=AudioTrack.HIGHWAY_NOCTURNE,
+            placeholder=AudioTrack.HIGHWAY_NOCTURNE,
+        )
+        voice: Voice = SchemaField(
+            description="AI voice to use for narration",
+            default=Voice.LILY,
+            placeholder=Voice.LILY,
+        )
+        video_style: VisualMediaType = SchemaField(
+            description="Type of visual media to use for the video",
+            default=VisualMediaType.STOCK_VIDEOS,
+            placeholder=VisualMediaType.STOCK_VIDEOS,
+        )
+
+    class Output(BlockSchema):
+        video_url: str = Field(description="The URL of the created video")
+        error: Optional[str] = Field(description="Error message if the request failed")
+
+    def __init__(self):
+        super().__init__(
+            id="361697fb-0c4f-4feb-aed3-8320c88c771b",
+            description="Creates a shortform video using revid.ai",
+            categories={BlockCategory.SOCIAL, BlockCategory.AI},
+            input_schema=AIShortformVideoCreatorBlock.Input,
+            output_schema=AIShortformVideoCreatorBlock.Output,
+            test_input={
+                "api_key": "test_api_key",
+                "script": "[close-up of a cat] Meow!",
+                "ratio": "9 / 16",
+                "resolution": "720p",
+                "frame_rate": 60,
+                "generation_preset": GenerationPreset.LEONARDO,
+                "background_music": AudioTrack.HIGHWAY_NOCTURNE,
+                "voice": Voice.LILY,
+                "video_style": VisualMediaType.STOCK_VIDEOS,
+            },
+            test_output=(
+                "video_url",
+                "https://example.com/video.mp4",
+            ),
+            test_mock={
+                "create_webhook": lambda: (
+                    "test_uuid",
+                    "https://webhook.site/test_uuid",
+                ),
+                "create_video": lambda api_key, payload: {"pid": "test_pid"},
+                "wait_for_video": lambda api_key, pid, webhook_token, max_wait_time=1000: "https://example.com/video.mp4",
+            },
+        )
+
+    def create_webhook(self):
+        url = "https://webhook.site/token"
+        headers = {"Accept": "application/json", "Content-Type": "application/json"}
+        response = requests.post(url, headers=headers)
+        response.raise_for_status()
+        webhook_data = response.json()
+        return webhook_data["uuid"], f"https://webhook.site/{webhook_data['uuid']}"
+
+    def create_video(self, api_key: str, payload: dict) -> dict:
+        url = "https://www.revid.ai/api/public/v2/render"
+        headers = {"key": api_key}
+        response = requests.post(url, json=payload, headers=headers)
+        logger.info(f"API Response Status Code: {response.status_code}")
+        logger.info(f"API Response Content: {response.text}")
```

Comment:
nit: merge these two logs ?

Reply:
also: these log statements are non-specific making them practically noise in production, and considering their content they should be `debug` not `info`

Reply:
Done